scripts/mini_RADI/Dockerfile.mini_humble

```diff
@@ -13,8 +13,8 @@ RUN mv /opt/jderobot/hal_interfaces /opt/jderobot/CustomRobots /opt/jderobot/jde
 
 # Compiling and sourcing the workspace
 WORKDIR /home/ws
-RUN /bin/bash -c "source /home/drones_ws/install/setup.bash; rosdep install --from-paths src --ignore-src -r --rosdistro humble -y"
-RUN /bin/bash -c "source /opt/ros/humble/setup.bash; colcon build --symlink-install"
+# RUN /bin/bash -c "source /home/drones_ws/install/setup.bash; rosdep install --from-paths src --ignore-src -r --rosdistro humble -y"
```

Comment:
instead of unsourcing, try to remove cloning repositories `as2_platform_pixhawk` and `as2_px4_msgs`. It might solve the issue while colcon-building